allensdk/brain_observatory/behavior/data_objects/metadata/subject_metadata/age.py

```diff
@@ -25,17 +31,34 @@ def from_json(cls, dict_repr: dict) -> "Age":
     @classmethod
     def from_lims(cls, behavior_session_id: int,
                   lims_db: PostgresQueryMixin) -> "Age":
+        # TODO PSB-17: Need to likewise grab the daq from the stimulus
+        # file for now as the data for daq in LIMS needs to be
+        # updated.
+        date_of_acquisition = DateOfAcquisition.from_stimulus_file(
+            BehaviorStimulusFile.from_lims(
+                db=lims_db,
+                behavior_session_id=behavior_session_id).validate()
+        ).value
+
         query = f"""
-            SELECT a.name AS age
+            SELECT d.date_of_birth AS date_of_birth
             FROM behavior_sessions bs
             JOIN donors d ON d.id = bs.donor_id
-            JOIN ages a ON a.id = d.age_id
             WHERE bs.id = {behavior_session_id};
         """
-        age = lims_db.fetchone(query, strict=True)
-        age = cls._age_code_to_days(age=age)
+        date_of_birth = cls._check_timezone(
+            lims_db.fetchone(query, strict=True))
+
+        age = (date_of_acquisition - date_of_birth).days
         return cls(age=age)
 
+    @classmethod
+    def _check_timezone(cls, input_date: datetime) -> datetime:
+        if input_date.tzinfo is None:
+            # Add UTC tzinfo if not already set
+            input_date = pytz.utc.localize(input_date)
```

Comment:
Is this done because it's assumed that date_of_acquisition is localized (PST?) ?

Reply:
Yes, matching the logic in the DateOfAcquiisition object [here](https://github.com/AllenInstitute/AllenSDK/blob/9ef5214dcb04a61fe4c04bf19a5cb13c9e1b03f1/allensdk/brain_observatory/behavior/data_objects/metadata/behavior_metadata/date_of_acquisition.py#L19).

Reply:
I see, it localizes it to UTC which makes more sense to me.